blueprints/modular/about.yaml

```
@@ -0,0 +1,30 @@
+title: About
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        features:
+          type: tab
+          title: About
+          fields:
+            header.details:
+              name: details
+              type: list
+              label: Details

```

Let's add a description of this field.
```suggestion
              label: Details
              description: Sub-sections to be shown after the page's Markdown content. Three details typically looks best.
```

---

Yeah I should have definitely put a description in here.